### PR TITLE
Make multiple choice more obvious

### PR DESCRIPTION
**Pull request by jamthief**
I think there might be more places that we could add this, to make it super obvious that the question requires ticking multiple answers. Please think of this commit as the start of a discussion - if agreed then I'm happy to go through and find others and update the PR.

### CODE REVIEW

**Comment by martinohanlon** [comment]
Thank you for letting us know.

Questions which have multiple answers should include instructions to let learners know more than 1 answer should be selected.

We tend to go with `(Select all that apply)` e.g.  https://github.com/neo4j-graphacademy/courses/blob/83ceb9a98d9080b4410f3e0fb8bea09bd75e07f0/asciidoc/courses/importing-fundamentals/modules/2-data-importer/lessons/8-data-importer-considerations/questions/1-data-importer.adoc#L4

Check and checkmarks are typically used only in the US. The same way ticks tend to be used in the UK.

If you find more, please do let us know or submit a PR, its super helpful.

File: asciidoc/courses/cypher-fundamentals/modules/1-reading/lessons/3-relationships/questions/2-directors.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [.question]
 = 2. Query using a pattern
 
-Which `MATCH` clauses will return the names of the directors of the movie, _The Matrix_?
+Which `MATCH` clauses will return the names of the directors of the movie, _The Matrix_? Check all that apply.
```
Comment (jamthief) [inline asciidoc/courses/cypher-fundamentals/modules/1-reading/lessons/3-relationships/questions/2-directors.adoc:4]:
```suggestion
Which `MATCH` clauses will return the names of the directors of the movie, _The Matrix_? (Select all that apply)
```

**Comment by jamthief** [comment]
Thanks @martinohanlon - updated!

**Review by martinohanlon — APPROVED** [review]

**Comment by martinohanlon** [comment]
Thanks @jamthief - Ill push when we do our next release.

**Pull request merged**